orchagent/tunneldecaporch.cpp

```diff
@@ -327,8 +389,8 @@ bool TunnelDecapOrch::addDecapTunnel(string key, string type, IpAddresses dst_ip
 
     tunnelTable[key] = { tunnel_id, overlayIfId, dst_ip, {} };
 
-    // create a decap tunnel entry for every ip
-    if (!addDecapTunnelTermEntries(key, dst_ip, tunnel_id))
+    // create a decap tunnel entry for every source_ip - dest_ip pair
+    if (!addDecapTunnelTermEntries(key, src_ip, dst_ip, tunnel_id, term_type))
```

Comment:
You can pass it as `*p_src_ip` and retain the existing variables.

Reply:
Yes, we can do that. But we have to check if p_src_ip is NULL. So I think adding another variable is better here.

Reply:
ok